core/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -250,7 +250,8 @@ private void fromHdfsToTachyon(TachyonURI path) throws IOException {
 
   @Override
   public long getDefaultBlockSize() {
-    return getConf().getLong("fs.local.block.size", Constants.DEFAULT_BLOCK_SIZE_BYTE);
+    return mTachyonConf.getBytes(Constants.USER_DEFAULT_BLOCK_SIZE_BYTE,
```

Comment:
This change will make Tachyon write the underFS with its default block size rather than the underFS's default block size?